resctrl/collector.go

```diff
@@ -26,19 +26,14 @@ import (
 )
 
 type collector struct {
-	resctrl intelrdt.IntelRdtManager
+	resctrl intelrdt.Manager
 	stats.NoopDestroy
 }
 
 func newCollector(id string, resctrlPath string) *collector {
 	collector := &collector{
```

Comment:
what do the changes here have to do with the runc update?

Reply:
@xiaochenshen made a new way of creating IntelRdt manager

https://github.com/opencontainers/runc/commit/933c4d31a77ac8b85982f960a09b7257fff51044
https://github.com/opencontainers/runc/commit/2c004a101ecd8b3b5604e53782ac1eb56859e4e5

Reply:
Sorry for the late reply -- the reason for the change was duly explained in the commit message.